README.md

```diff
@@ -33,8 +33,11 @@ On which edge of the editor should the tool bar appear. Possible options:
 ### Icon size
 
 *   Very Small *(12px)*
-*   Small *(16px)*
+*   Smaller *(16px)*
+*   Small: *(18px)*
```

Comment:
Perhaps change this to :arrow_down:?

```markdown
*   Tiny *(12px)*
*   Vary small *(16px)*
*   Small: *(18px)*
```

As far as I know changing these names doesn't break backwards compatibility.

Reply:
Fixed in https://github.com/suda/tool-bar/pull/277/commits/3489f07427c703faaf7f2425a6a41c94dc82923d